src/gretel_trainer/relational/core.py

```diff
@@ -118,7 +125,9 @@ class RelationalData:
     See the specific method docstrings for details on each method.
     """
 
-    def __init__(self):
+    def __init__(self, directory: Optional[Union[str, Path]] = None):
+        self.dir = Path(directory or DEFAULT_RELATIONAL_SOURCE_DIR)
+        os.makedirs(self.dir, exist_ok=True)
```

Comment:
Same option as above
```suggestion
        self.dir.mkdir(parents=True, exist_ok=True)
```